### ISSUE TITLE
No more decimals in Total values since last update

### ISSUE THREAD

**Issue by MTRKhn** [biemond/solaredge.modbus#28]
Total yield, Total export and total import have no decimal values anymore so calculating a daily consumption is not precise anymore... Is it possible to revert that change please ?

**Comment by biemond** [comment]
Hey,

will do. 
by the way is the daily total yield not ok.   if not can you show me your calculation for this. Maybe I can add this.

thanks

**Comment by MTRKhn** [comment]
Nope, Total Day Yield is not correct (or i generated 152kWh yesterday lol).

I'm just collecting the "meter-power" information in InfluxDb and then representing using Grafana for the desired period. Well, i'm collecting and representing a lot of SolarEdge data :D 

![image](https://user-images.githubusercontent.com/72077818/212025962-e8d80807-67ab-4300-9cf7-93e3be88f54a.png)

**Comment by biemond** [comment]
that part is fixed , in NL they use 00:00 as midnight , others 24:00. 
can you try this https://homey.app/a/solaredge.modbus/test/ and on friday it should work.

**Comment by biemond** [comment]
I will add the decimals back.

**Comment by MTRKhn** [comment]
> that part is fixed , in NL they use 00:00 as midnight , others 24:00. can you try this https://homey.app/a/solaredge.modbus/test/ and on friday it should work.

I already installed .15 to check in the decimal issue was fixed. I'll update this thread tomorrow :)

**Comment by MTRKhn** [comment]
> that part is fixed , in NL they use 00:00 as midnight , others 24:00. can you try this https://homey.app/a/solaredge.modbus/test/ and on friday it should work.

I checked this morning. "Total Day Yield" is 1.48kWh but SolarEdge app reports 2.06kWh :(

**Comment by biemond** [comment]
yep I need to calculate this [ ](https://github.com/biemond/solaredge.modbus/blob/main/drivers/solaredge.ts#L277)
maybe solaredge has more info then me.

**Comment by biemond** [comment]
https://github.com/biemond/solaredge.modbus/blob/main/drivers/solaredge.ts#L277

**Comment by biemond** [comment]
published a new test version with 2 decimals

**Comment by MTRKhn** [comment]
All good now. Thanks !
